src/web.cpp

```diff
@@ -752,17 +752,16 @@ void handleRoot() {
         const char* off = "Off";
 
         if (ConfigSettings.connectedClients > 0) {
-            if (ConfigSettings.connectedClients > 1) {
-                doc[connectedSocketStatus] = "Yes, " + String(ConfigSettings.connectedClients) + "connection";
-                doc[connectedSocket] = readableTime;
-            } else {
-                doc[connectedSocketStatus] = "Yes, " + String(ConfigSettings.connectedClients) + " connections";
-                doc[connectedSocket] = readableTime;
-            }
+            char str[32];
+            int n = ConfigSettings.connectedClients;
```

Comment:
I think it is better to make the variable "n" a constant. The int type is redundant, uint8_t is better

Reply:
> Do you mean to suggest I make this const int

`const uint8_t`

> I can go change the connectedClients in config.h from int to uint8_t to save a few bytes of memory

Yes, that would be great.

> Looking at the emitted assembly, it uses 32-bit general purpose registers, anyway.

That's strange, I always thought that specifying a var type should work in the arduino framework